packages/app-studio-toolkit/src/public-api/create-workspace-proxy.ts

```diff
@@ -12,6 +12,8 @@ export function createWorkspaceProxy(
     onWorkspaceChanged: (
       ...args: Parameters<WorkspaceApi["onWorkspaceChanged"]>
     ) => workspaceImpl.onWorkspaceChanged(...args),
+    startWatch: () => workspaceImpl.startWatch(),
```

Comment:
i don't think we should expose this methods.
What happens if one BAS extension does "stopWatch"?
I assume it would "stop" watching for **all** BAS extensions
using this singleton project-api object...

Reply:
I totally agree with you. Thanks for correcting me and I will revert this commit.